### ISSUE TITLE
[Windows 7] 'gcc' could not be spawned when gcc is in PATH

### ISSUE THREAD

**Issue by charlw** [kriscross07/atom-gpp-compiler#116]
Just installed Atom + gpp-compiler to try a fresh IDE. I'm in the process of trying to run a simple hello world C script to test Atom, but Atom is throwing an error suggesting it cannot find gcc. 

When I hit F5 to Compile and Run, I get the following:
> 'gcc' could not be spawned. Is it installed and on your path? If so please open an issue on the package spawning the process.

Currently have gcc in PATH as follows:
`C:\Users\Charlie\AppData\Roaming\npm;C:\TDM-GCC-64\bin`

The error console is (roughly, excuse the lack of proper indentation) displaying:

>Uncaught Error: spawn gcc ENOENT                                                             events.js:160 
>exports._errnoException                         @ util.js:1026
>ChildProcess._handle.onexit                   @ internal/child_process.js:193
>onErrorNT                                               @ internal/child_process.js:359
>_combinedTickCallback                          @ internal/process/next_tick.js:74
>_tickCallback                                            @ internal/process/next_tick.js:98

Typing `gcc --version` in cmd returns:
>gcc (tdm64-1) 5.1.0 
>Copyright (C) 2015 Free Software Foundation, Inc.
>This is free software; see the source for copying conditions. There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 

It is possible for me to open console, `cd` to a C script file path, and type `gcc <scriptname>.c` to execute it.

I also have MinGW on my system and have attempted to run C scripts on Atom with it by specifying MinGW's location in PATH and changing gcc to MinGW in the atom-gpp-compiler preferences, but I received the same 'could not be spawned' issue for MinGW.

**Comment by turnipjs** [comment]
Have you restarted Atom since adding to your path?

**Comment by charlw** [comment]
Yep - went through standard debugging procedure back when this was an issue.

As this is several months since I 'fixed' it, I believe (but might be wrong) that the issue was that I had entered the full filepath to GCC rather than simply putting literally 'gcc' as the C Compiler in the gpp compiler settings.
